### ISSUE TITLE
Enhance terminal muting functionality to play bell sound only once

### ISSUE THREAD

**Issue by MaxRis** [jumptrading/FluentTerminal#102]
Initial issue for this in upstream https://github.com/felixse/FluentTerminal/issues/386

UPDATED:

- Tray process on startup searches for available conhost audio sessions. If no audio session is available, detached cmd.exe process is spawned to play bell sound. Tray process searches for conhost audio session again and sets relevant mute state.
- Since detached cmd.exe process spawned on first step with its active audio session will seat active in background (need to investigate if audio session is not get expired automatically or just disappears for some reasons), Fluent will not generate more bell sounds on next launches or on change of Mute Terminal option value in Settings.

UPDATE [2019.07.19]:

As it was discussed with @mjs it makes sense to try to update current implementation to mute generated bell audio session as soon as possible after the start. Potentially, it might significantly simplify the solution.

**Comment by MaxRis** [comment]
Discussion takes place in upstream Issues thread https://github.com/felixse/FluentTerminal/issues/386

**Comment by MaxRis** [comment]
PR into upstream https://github.com/felixse/FluentTerminal/pull/428

**Comment by MaxRis** [comment]
Resolved in our fork by PR https://github.com/jumptrading/FluentTerminal/pull/159

As it was discussed with @mjs it makes sense to try to update current implementation to mute generated bell audio session as soon as possible after the start. Potentially, it might significantly simplify the solution.

**Comment by MaxRis** [comment]
@mjs new updated implementation to mute generated bell sound as fast as possible is merged into our fork with https://github.com/jumptrading/FluentTerminal/pull/164

Unfortunately, there is no API to catch start of C# Process async ( https://github.com/jumptrading/FluentTerminal/pull/164/files#diff-cd75e42d7c72afd7283483cd9a6cd982R621 ), but it seems it just fine to start it synchronously.

Essential hack was to delay generation of bell sound to 1 second ( by running [cmd.exe /k timeout 1](https://github.com/jumptrading/FluentTerminal/pull/164/files#diff-cd75e42d7c72afd7283483cd9a6cd982R620)) and allow Fluent [to find](https://github.com/jumptrading/FluentTerminal/pull/164/files#diff-cd75e42d7c72afd7283483cd9a6cd982R635) corresponding conhost.exe by parent process id.

Could you please give feedback, thanks!

**Comment by mjs** [comment]
It looks like this was merged upstream in https://github.com/felixse/FluentTerminal/pull/468. It works perfectly with no muting of other sounds. Nicely done.
